client/package.json

```diff
@@ -9,6 +9,7 @@
     "build-local": "vue-tsc --noEmit && vite build --mode staging",
     "serve": "vite preview --host",
     "preview": "vue-tsc --noEmit && vite build --mode staging && vite preview --host",
+    "preview-8000": "vue-tsc --noEmit && vite build --mode staging && vite preview --host --port 8000",
```

Comment:
Wozu nochmal ein Preview?

Reply:
Dann kann man cypress mit einem build testen. Denn der normale ist auf port 4173. Und cypress lokal schaut auf 8000

Reply:
Verstehe ich gerade nicht. Müssen wir aber nicht mehr in den nächsten drei Tagen klären ;-)